sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -860,12 +860,23 @@ def _add_argparse_args(cls, parser):
             'https://cloud.google.com/compute/docs/vpc/'))
     parser.add_argument(
         '--worker_harness_container_image',
```

Comment:
Should this usage be marked with a user visible deprecation warning?

Reply:
I think because we are mostly keeping this around for legacy pipelines, it's not fully deprecated. I can edit the message though.